wire-ios-request-strategy/Sources/Helpers/EncryptionSessionDirectory+UpdateEvents.swift

```diff
@@ -20,7 +20,7 @@ import Foundation
 import WireSystem
 import WireCryptobox
 
-private let zmLog = ZMSLog(tag: "cryptobox")
+private let zmLog = ZMSLog(tag: "proteus")
```

Comment:
I wouldn't change this, we only want to change the name for the new logs and not the existing ones.